applications/IceTrays.md

```diff
@@ -0,0 +1,136 @@
+
+  ​	IceTrays cluster based on raft consensus, combined with IPFS and CRUST technology, combines multiple IPFS nodes into a cluster to provide users with hot storage, while crust network is used as cold storage to ensure reliability. IceTrays node supports both file path and CID retrieval, which provides an easy-to-use  combined API for IPFS and CRUST.
+
+  ​	 I have written a general framework to familiarize  with the raft consensus. The [minimum viable product](https://github.com/ysh0566/ipfs-fs-cluster) can add, move, delete file and make directory, and  other nodes in network can sync directory root state from leader. Recently I noticed the crust project. I think crust is well integrated with it, which gives me motivation to improve the project.
+
+  ​	 It's a golang project.
+
+  
+
+  ### Ecosystem Fit 
+
+  Help us locate your project in the Crust landscape and what problems it tries to solve by answering each of these questions:
+
+  * Where and how does your project fit into the ecosystem? 
+
+    Decentralized Pinning Service
+
+  * What need(s) does your project meet? 
+
+    Providing  Helping users to pin their files in Crust network.
+
+  * Are there any other projects similar to yours in the IPFS/Crust ecosystem? 
+
+    [Powergate](https://github.com/textileio/powergate) is a multitiered file storage API built on Filecoin and IPFS, and an index builder for Filecoin data. IceTrays provides a more reliable  and easy-to-used service because it is decentralized and its nodes store file path message.
+
+  ## Team
+
+  ### Team members
+
+  * Shenghui Yu
+  * Shiwei Huang	
+
+  ### Contact
+
+  * **Contact Name:** Shenghui Yu
+  * **Contact Email:** ysh0566@qq.com
+  * **Website:** https://github.com/ysh0566
+
+  ### Legal Structure 
+
+  none
+
+  ### Team's experience
+
+  We are from a blockchain technology team. Our work a year ago was related to public chain development, and our work in the latest year was related to filecoin. We participated in slingshot and won the [community call-out award](https://github.com/filecoin-project/slingshot/blob/master/participants/ChainsData.md).
+
+  ### Team Code Repos
+
+  * https://github.com/IceTrays
+
+  ### Team LinkedIn Profiles
+
+  None
+
+  ## Development Roadmap
+
+  ### Overview
+
+  * **Total Estimated Duration:** 3 months
+  * **Full-Time Equivalent (FTE):**  2 FTE
+  * **Total Costs:**  $24k
+
+  ### Milestone 1 — Implement a minimum viable product
+
+  * **Estimated Duration:** 6 weeks (Keep two extra weeks to learn about crust ecology)
+  * **FTE:**  2
+  * **Costs:** 10, 000 USD
+
+  | Number | Deliverable              | Specification                                                |
+  | -----: | ------------------------ | ------------------------------------------------------------ |
+  |    0a. | License                  | MIT license                                                  |
```

Comment:
License is not make any sense to me as a single work item

Reply:
just copy from application_template.md......

Reply:
Then, delete it.